src/components/SuggestionsInput.react.js

```diff
@@ -0,0 +1,598 @@
+                                ? `${getComputedNumStyleAttr(
+                                      this._input,
+                                      'height'
+                                  )}px`
+                                : '16px'
+                        }
+                        leftOffset={
+                            this._hiddenValue
+                                ? `${(() => {
+                                      const hiddenWidth = getComputedNumStyleAttr(
+                                          this._hiddenValue,
+                                          'width'
+                                      );
+                                      const cursorPos = this._input
+                                          .selectionStart;
+                                      // TODO consider textarea lines
+                                      const r =
+                                          (hiddenWidth / value.length) *
+                                          cursorPos;
+
+                                      const maxWidth = getComputedNumStyleAttr(
+                                          this._input,
+                                          'width'
+                                      );
+
+                                      if (r > maxWidth) {
+                                          return maxWidth;
+                                      }
+
+                                      return r;
+                                  })()}`
+                                : 0
+                        }
+                        onSuggestion={this.onSuggestion}
+                        style={suggestions_style}
+                        className={suggestions_className}
+                        suggestion_style={suggestion_style}
+                        suggestion_className={suggestion_className}
+                        selected_style={suggestion_selected_style}
+                        selected_className={suggestion_selected_className}
+                    />
+                )}
+                <div
+                    ref={r => (this._hiddenValue = r)}
+                    style={merge(style, {
+                        display: 'inline-block',
+                        visibility: 'hidden',
+                    })}
+                >
+                    {this.state.value}
+                </div>
+            </div>
+        );
+    }
+}
+
+SuggestionsInput.defaultProps = {
+    multi_line: false,
+    value: '',
+    allow_space_in_suggestions: false,
+    triggerless: false,
+};
+
+const OptionsShape = PropTypes.shape({
+    value: PropTypes.string.isRequired,
+    display: PropTypes.string,
+    description: PropTypes.string,
+});
+
+SuggestionsInput.propTypes = {
+    id: PropTypes.string,
+    /**
+     * Css class for the container of the input and suggestions modal.
+     */
+    className: PropTypes.string,
+    /**
+     * Style object given to the topmost container.
```

Comment:
is topmost container the same node as the target above "container of the input and suggestions modal"? If so, let's make the descriptions of the target node the same.